src/test/resources/test_data/2013_minneapolis_park/2013_minneapolis_park_expected.json

```diff
@@ -240,7 +240,7 @@
   }, {
     "round" : 11,
     "tally" : {
-      "ANNIE YOUNG": "14866",
+      "ANNIE YOUNG" : "14866",
```

Comment:
this is super weird

Reply:
interestingly the new test fails for me as a result of whitespace formatting differences.

Reply:
Finally figured this out. See the discussion in #232. Thanks, IntelliJ.